ignite/handlers/param_scheduler.py

```diff
@@ -1393,6 +1393,99 @@ def simulate_values(cls, num_events: int, schedulers: List[_LRScheduler], **kwar
             return values
 
 
+class ReduceLROnPlateauScheduler(ParamScheduler):
+    """Reduce LR when a metric stops improving.
+    Wrapper of torch.optim.lr_scheduler.ReduceLROnPlateau
+    <https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.ReduceLROnPlateau.html>.
+
+    Args:
+        optimizer: Wrapped optimizer.
+        metric_name: metric whose improvement is monitored.
+            Must be attached the to same engine.
+        trainer: Trainer engine to log LR history in its
+            `state.output.param_history`. Is used if `save_history`
+            is true. Default: None.
+        save_history: Whether to save history or not. If true,
+            history will be logged in `trainer`'s `state.output.param_history`.
+            Default: False.
+        param_group_index: `optimizer`'s parameters group
+            to use.  Default: None. Use all `optimizer`'s paramater groups.
+        **scheduler_kwargs: Keyword arguments to be passed to the wrapped
```

Comment:
Just a note: torch scheduler has `verbose` arg, but in our case, it seems like it wont be taken into account...
I'm not sure if it is good to use `print` as torch is doing now...

Reply:
I hard-coded it to `False` internally. We have `save_param_history` ourselves.

Reply:
Where it is done exactly ?

Reply:
Now I committed.

Reply:
Let's also mention that in the docs: verbose is set to False and user can use `save_history`.